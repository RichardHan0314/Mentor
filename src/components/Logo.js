import React from 'react';
import styled from 'styled-components';

const Logo = () => {
    return (
        <LogoName><span>Netflix</span>roulette</LogoName>
    );
};

const LogoName = styled.p`
    color: #f65261;
    font-size: 20px;
    margin: 0;

    span {
        font-weight: bold;
    }
`;

export default Logo;
