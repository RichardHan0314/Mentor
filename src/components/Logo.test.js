import React from 'react';
import { render, screen } from '@testing-library/react';
import Logo from './Logo';

describe('Logo component', () => {
    it('Render Logo snapshot', () => {
        const { asFragment } = render(<Logo />);
        expect(asFragment(<Logo />)).toMatchSnapshot();
    });

    it('Render Logo correctly', () => {
        render(<Logo />);
        // screen.debug();
        expect(screen.getByText('Netflix')).toBeInTheDocument();
    });
});
