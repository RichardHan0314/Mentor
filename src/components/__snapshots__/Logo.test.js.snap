// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component Render Logo snapshot 1`] = `
<DocumentFragment>
  <p
    class="sc-AxjAm ekgtgp"
  >
    <span>
      Netflix
    </span>
    roulette
  </p>
</DocumentFragment>
`;
